tox.ini

```diff
@@ -52,6 +52,7 @@ commands =
     nosetests --with-timer --rednose -sv contrib/runners/mistral_v2/tests/unit/ contrib/runners/mistral_v2/tests/integration/
     nosetests --with-timer --rednose -sv contrib/runners/python_runner/tests/unit/ contrib/runners/python_runner/tests/integration/
     nosetests --with-timer --rednose -sv contrib/runners/windows_runner/tests/unit/
+    nosetests --with-timer --rednose -sv contrib/runners/winrm_runner/tests/unit/
```

Comment:
Thanks for adding this here to make sure we also run new tests under Python 3.

It looks like pywinrm also works under Python 3 so that's good as well :+1: